en/development/errors.rst

```diff
@@ -64,7 +65,7 @@ deprecated. We also recommend this system for use in your plugins and
 application code when useful. You can trigger deprecation warnings with
 ``deprecationWarning()``::
 
-    deprecationWarning('5.0', 'The example() method is deprecated. Use getExample() instead.');
+    deprecationWarning('The example() method is deprecated. Use getExample() instead.');
```

Comment:
Version no. is needed as 1st argument in 5.x.

Reply:
Looks like a couple reverts didnt take.

Reply:
Fixed this and removed references to shell/task exceptions. The remaining changes are from 4.x.